guides/common/modules/con_planning-project-upgrade.adoc

```diff
@@ -0,0 +1,28 @@
+[id="planning-{project-context}-upgrade"]
+= Planning {Project} upgrade
+
+Upgrading to {Project} {ProjectVersion} affects your entire {Project} infrastructure.
+Plan carefully before proceeding.
+
+* Read the {ProjectName} {ProjectVersion} {ReleaseNotesDocURL}[Release Notes].
+* Consider whether any of your scripts need updating.
+Some {Project} API endpoints, Hammer CLI commands, and modules from the {Project} Ansible Collection can differ between versions of {Project}.
+ifdef::satellite[]
+For information about changes in these tools, see the {ProjectName} {ProjectVersion} {ReleaseNotesDocURL}[Release Notes].
+endif::[]
+ifdef::satellite[]
+* Optional: You can test the upgrade on a clone of your {ProjectServer}.
+After you successfully test the upgrade on the clone, you can repeat the upgrade on your primary {ProjectServer} and discard the clone, or you can promote the clone to your primary {ProjectServer} and discard the previous primary {ProjectServer}.
+For more information, see {AdministeringDocURL}cloning_satellite_server[Cloning {ProjectServer}] in _{AdministeringDocTitle}_.
+endif::[]
+ifndef::foreman-deb[]
+* If your {ProjectServer} runs on a RHEL system that is not operating in FIPS mode, you cannot upgrade it to a RHEL system that is operating in FIPS mode.
+In this situation, you must install {Project} on a freshly provisioned RHEL system in FIPS mode.
+endif::[]
```

Comment:
while technically true, I always thought it doesn't belong into the upgrade docs. the user also can't enable fips after the fact during normal operations.

I recall we wanted to have a "security considerations" guide, that's where it belongs.

Reply:
At first I thought this was the result of a customer case request. But actually it was part of the initial import: https://github.com/theforeman/foreman-documentation/commit/84e812e1326 so it's very old.
And I do think that we have this covered in the installation guide: https://github.com/theforeman/foreman-documentation/blob/028b74684e59bcb042835d20da4713e095cd67ed/guides/common/modules/ref_system-requirements.adoc?plain=1#L107-L115, which is being reviewed by Jirka in https://github.com/theforeman/foreman-documentation/pull/3799 in an effort to turn it into a "security considerations" section somewhere in the docs set.

Overall, I think we can remove this.

Reply:
Some more archeology with zombies later: this was added as part of https://bugzilla.redhat.com/show_bug.cgi?id=1671445 (the bug itself doesn't contain much data, sorry) as in Satellite 6.5 FIPS was a new feature, but users could not go from a non-FIPS 6.4 to a FIPS 6.5, and that's what this was warning against.

My calendar says this is safe to remove from the upgrade docs indeed.